readme.md

```diff
@@ -5,37 +5,52 @@
   <a href="https://prettier.io"><img alt="Prettier Code Formatting" src="https://img.shields.io/badge/code_style-prettier-brightgreen.svg" /></a>
 </p>
 
-# Dexter <!-- omit from toc -->
+# Dexter
 
-Dexter is a set of mature LLM tools used in production at [Dexa](https://dexa.ai), with a focus on real-world RAG ([Retrieval Augmented Generation](https://arxiv.org/abs/2005.11401)).
+Dexter is a powerful TypeScript library for working with Large Language Models (LLMs), with a focus on real-world Retrieval-Augmented Generation (RAG) applications. It provides a set of tools and utilities to interact with various AI models, manage caching, handle embeddings, and implement AI functions.
 
-_If you're a TypeScript AI engineer, check it out!_ 😊
+## Features
```

Comment:
(nit) this section has too much prose; the whole point of features is to give an at-a-glance overview for devs who haven't decided if they even care yet.

(nit = feel free to ignore, but I wanted to comment w/ feedback anyhow)

Reply:
This is what happens when you let an LOM write your docs 😅